lib/pages/chat/events/reply_content_style.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:fluffychat/config/app_config.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+class ReplyContentStyle {
+  static const double fontSizeDisplayName = AppConfig.messageFontSize * 0.76;

```

Comment:
can you change it like the design in figma

Reply:
It was implemented by fluffy, @hoangdat can you confirm we don't use `AppConfig.messageFontSize` anymore pls ?